### PR TITLE
Migrating docs from Docusarus to GQLAlchemy GitHub repo

### PR DESCRIPTION
**Pull request by kgolubic**
### Description

We will migrate the docs from Docusarus (https://memgraph.com/docs/gqlalchemy/) to GitHub which belongs to the GQLAlchemy repo.

### Pull request type

Please delete options that are not relevant.

- [x] Documentation content changes

### Related issues

Delete section if this PR doesn't resolve any issues. 

Closes (link to issue)

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

######################################

### Reviewer checklist (the reviewer checks this part)
- [ ] Core feature implementation
- [ ] Tests
- [ ] Code documentation
- [ ] Documentation on [memgraph/docs](https://github.com/memgraph/docs)

######################################

### CODE REVIEW

**Comment by kgolubic** [comment]
@katarinasupe, just a quick note. I've started working on this. We already have a [reference folder](https://github.com/memgraph/gqlalchemy/tree/main/docs/reference/gqlalchemy) in this repo, so we will need to see how it relates to one in the [Memegraph repo folder ](https://github.com/memgraph/docs/tree/master/gqlalchemy/reference).

**Comment by katarinasupe** [comment]
Awesome @kgolubic :) The reference folder here should consist of the same content as the one in the Memgraph repository, so I think you can just ignore that folder in this migration 😄

**Comment by kgolubic** [comment]
> Awesome @kgolubic :) I think the reference folder here has the same content as the one in the Memgraph repository, so you can just ignore that folder in this migration 😄

That this is when it comes to migrating existing docs. We now need to build and link the docs.

File: README.md [hunk 1]
```diff
@@ -279,7 +86,7 @@ poetry run pytest . -k "not slow and not extras"
 
 ## Documentation
 
-The GQLAlchemy documentation is available on [memgraph.com/docs/gqlalchemy](https://memgraph.com/docs/gqlalchemy/).
+The GQLAlchemy documentation is available on [GitHub](https://github.com/memgraph/gqlalchemy).
 
 The documentation can be generated by executing:
```
Comment (katarinasupe) [inline README.md:91]:
```suggestion
The reference guide can be generated by executing:
```

File: README.md [hunk 2]
```diff
@@ -279,7 +86,7 @@ poetry run pytest . -k "not slow and not extras"
 
 ## Documentation
 
-The GQLAlchemy documentation is available on [memgraph.com/docs/gqlalchemy](https://memgraph.com/docs/gqlalchemy/).
+The GQLAlchemy documentation is available on [GitHub](https://github.com/memgraph/gqlalchemy).
```
Comment (katarinasupe) [inline README.md:90]: I would add information on building the docs with mkdocs and testing locally for our devs. I would say that the reference guide is generated from the code while other parts of the docs are written and located at docs directory. To test the documentation locally, do _this this and that_

**Review by katarinasupe — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (kgolubic) [reply README.md:90]: I'll add this to the README.

Comment (kgolubic) [reply README.md:90]: @katarinasupe I've added the instructions.

**Review by katarinasupe — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -279,14 +86,20 @@ poetry run pytest . -k "not slow and not extras"
...
+The reference guide can be generated from the code by executing:
 ```
 pip3 install pydoc-markdown
 pydoc-markdown
 ```
 
+Other parts of the documentation are written and located at docs directory. To test the documentation locally execute:
+```
+pip3 install mkdocs
+mkdocs serve
```
[the remark this answers is not in the document]
Comment (kgolubic) [reply README.md]: Added to the file along with another dependency.

**Pull request merged**